xla/service/gpu/BUILD

```diff
@@ -3251,10 +3250,8 @@ cc_library(
         "@com_google_absl//absl/log:check",
         "@com_google_absl//absl/status",
         "@com_google_absl//absl/status:statusor",
-        "@com_google_absl//absl/strings",
         "@com_google_absl//absl/strings:string_view",
         "@local_config_cuda//cuda:cudnn_header",
-        "//xla:shape_util",
```

Comment:
This line needs to stay I think, provides `primitive_util.h`

Reply:
Done.